build.gradle

```diff
@@ -54,7 +54,7 @@ apply plugin: 'org.jetbrains.dokka'
 apply plugin: 'com.google.protobuf'
 apply plugin: 'com.github.ben-manes.versions'
 apply plugin: 'com.github.dcendents.android-maven'
-apply plugin: 'com.getkeepsafe.dexcount'
+//apply plugin: 'com.getkeepsafe.dexcount'
```

Comment:
I don't think we should commit these changes as part of this PR.

I only commented them out because I was in a hurry and the plugin was having a problem and it's not necessary for building. We can correct or remove this plugin as part of a separate issue.

For now, we can leave these as local changes and reset this `build.gradle` file to keep it out of the PR.

Reply:
Reverted to initial grandle state

Reply:
It appears that you may have reverted a bit too far because instead of having no changes to the `build.gradle` file, this PR now has [changes](https://github.com/zcash/zcash-android-wallet-sdk/pull/206/commits/023d1fb079e9a719c765a1c1a549cbad4ed36ee1#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7) to undo the [two most recent commits to that file](https://github.com/zcash/zcash-android-wallet-sdk/commit/9bf087bed801cec75421ba22727f05944081e89e#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7).